samples/extensions/graphics_pipeline_library/README.adoc

```diff
@@ -144,6 +145,23 @@ image::./images/sample.jpg[Sample]
 This sample demonstrates that functionality by creating the shared vertex input interface, pre-rasterization shader state and fragment output interface parts only once up-front, and then re-uses them to create pipelines with customized fragment shaders using random lighting models at runtime.
 Pipelines are created in a background thread and once they're created, command buffers are updated to display a mesh using the new pipeline.
 
+== Independent Descriptor Sets
+
+While this sample doesn't use it, this extension has a `VK_PIPELINE_LAYOUT_CREATE_INDEPENDENT_SETS_BIT` flag that can be used.
```

Comment:
Should probably have _EXT at the end of VK_PIPELINE_LAYOUT_CREATE_INDEPENDENT_SETS_BIT. Nothing appears if you just search for VK_PIPELINE_LAYOUT_CREATE_INDEPENDENT_SETS_BIT with _EXT.

Reply:
good catch, updated